Packs/PerplexityAI/Integrations/PerplexityAI/PerplexityAI.yml

```
@@ -0,0 +1,92 @@
+category: Utilities
+commonfields:
+  id: Perplexity AI
+  version: -1
+configuration:
+- defaultvalue: https://api.perplexity.ai
+  display: URL for Perplexity AI
+  name: url
+  required: true
+  section: Connect
+  type: 0
+- display: Perplexity AI API Key
+  displaypassword: Perplexity AI API Key
+  hiddenusername: true
+  name: apikey
+  required: true
+  section: Connect
+  type: 9
+- display: Cloudflare Access Client Id
+  displaypassword: Cloudflare Access Client Secret
+  name: cf_auth
+  required: false
+  section: Connect
+  type: 9
+- additionalinfo: 'Available models are: sonar,sonar-pro,sonar-deep-research,sonar-reasoning,sonar-reasoning-pro,r1-1776'
+  defaultvalue: sonar-deep-research
+  display: Perplexity AI Model
+  name: model
+  required: true
+  type: 0
```

```suggestion
  type: 0
- display: Trust any certificate (not secure)
  name: insecure
  type: 8
  section: Connect
  advanced: true
  required: false
- display: Use system proxy settings
  name: proxy
  type: 8
  section: Connect
  advanced: true
  required: false
```
Please add the insecure and proxy paramerters
For more info [see](https://xsoar.pan.dev/docs/tutorials/tut-integration-ui#basic-integration-settings)